docs/faq/network/validator.md

```diff
@@ -0,0 +1,114 @@
+
+### What will be the different validator clients?
+
+At launch, the LUKSO CLI is able to install the following clients for running a validator node. More will become available in the future.
+
+- Execution Clients: [Geth](https://geth.ethereum.org/)
+- Consensus Clients: [Prysm](https://github.com/prysmaticlabs/prysm)
+
+### How can I set my withdrawal address for my validators rewards after the Shapella upgrade?
+
+For details on how to enable withdrawals after the Shapella upgrade see these guides:
+
+- [https://notes.ethereum.org/@launchpad/withdrawals-guide](https://notes.ethereum.org/@launchpad/withdrawals-guide)
+- [https://docs.prylabs.network/docs/wallet/withdraw-validator](https://docs.prylabs.network/docs/wallet/withdraw-validator)
+
+### Will lower latency positively influence my rewards?
+
+When the appointed time for a validator comes to propose a block the validator has 12s to vote. 2 scenarios could cause a validator to miss that slot:
+
+1. The node is not synced up to the canonical head of the network.
+2. The validator needed more than 12s to propose the block.
+
+There are no extra rewards for being very quick.
+
+### How can I set up my node to maximize my peers?
+
+In the normal case a node will connect to the maximum peers. Each new peer connection goes through an evaluation step that will judge if a node is suitable, otherwise it will disconnect. You can check the logs to see disconnection from peers.
+
+Max peers can be configured in the configuration of your node, however more peers increase bandwidth and do not necessarily lead to benefits.
+
+### Why is there a difference between my peers on the execution stats and consensus stats website?
+
+The consensus client follows the beacon chain and will sync by itself only to the final epoch. This means that once finality is reached it is not necessarily important how many peers the execution node has. The execution and consensus peers are also not necessarily the same. There is no mechanism where the consensus tells the execution node which peers it is connected to and vice versa.
+
+### What ports must be open on a node?
+
+The following ports will need to be opened to run your node correctly:
+
+```
+tcp:30303
+tcp:13000
+udp:12000
+udp:30303
+```
+
+### How can I open my ports on my node machine?
+
+The LUKSO CLI will automatically open the required ports for your node. This is due to the docker-compose configuration running in "host" mode. If you have a firewall configured, you should enable incoming traffic for the above mentioned ports. Ports will also need to be forwarded in your router.
```

Comment:
The current CLI does not use Docker and Docker Compose anymore. This section has to be reworked